packages/react-combobox/src/components/Combobox/useCombobox.ts

```diff
@@ -11,18 +23,167 @@ import type { ComboboxProps, ComboboxState } from './Combobox.types';
  * @param props - props from this instance of Combobox
  * @param ref - reference to root HTMLElement of Combobox
  */
-export const useCombobox_unstable = (props: ComboboxProps, ref: React.Ref<HTMLElement>): ComboboxState => {
+export const useCombobox_unstable = (
+  props: ComboboxProps,
+  optionCollection: OptionCollectionState,
+  ref: React.Ref<HTMLButtonElement>,
+): ComboboxState => {
+  const {
+    inline = false,
+    multiselect,
+    onOpenChange,
+    open: controlledOpen,
+    placeholder,
+    positioning,
+    value: controlledValue,
+  } = props;
+  const {
+    options,
+    collectionData: { count, getOptionAtIndex, getIndexOfKey, getOptionByKey },
+  } = optionCollection;
+  const idBase = useId('combobox');
+
+  const [activeOption, setActiveOption] = React.useState<OptionValue | undefined>();
+  const [selectedKeys, selectKey] = useSelection(props);
+
+  const [value, setValue] = useControllableState({
```

Comment:
```suggestion
  const [value, setValue] = useControllableState({
    defaultValue: props.value,
```

This should also have `default*` prop to be autocontrolled